packages/v4/src/content/design-guidelines/components/label/label.md

```diff
@@ -3,60 +3,75 @@ id: Label
+
+Use the filled option to add more visual prominence to a label. If your use case requires a mix of clickable and non-clickable labels, consider using one type for clickable and another for non-clickable. Whatever convention you choose, be sure to maintain consistency through your UI.
+
+1. Unfilled
+
+<img src="./img/unfilled-labels.png" alt="Unfilled labels" width="646"/>
+
+2. Filled
+
+<img src="./img/filled-labels.png" alt="Filled labels" width="646"/>
+
+Use the filled option to add more visual prominence to a label.
 
 ### When to use dismissable labels
 
-For labels that are easily removed or dismissed, use a dismissable label. A common use case for dismissable labels is when users can edit labels.
+Use a dismissable label for labels that are easily removed. A common use case for dismissable labels is when users can edit labels.
 
 <img src="./img/edit-labels.png" alt="Example of dismissable labels" width="646"/>
 
 ### When to use labels with icons
-
-Optionally add an icon to your label in cases where additional visual information is helpful.
+Add an icon to your label in cases where additional visual information is helpful. You can also use them to distinguish labels of the same color if needed.
 
 <img src="./img/labels-with-icons.png" alt="Example of different colored labels with icons" width="318"/>
 
-You can also use them to distinguish labels of the same color if needed.
-
-<img src="./img/labels-with-icons2.png" alt="Example of labels of the same color with icons" width="320"/>
-
 ### When to use clickable labels
+Use a clickable label for actionable labels. A common use case for clickable labels is when a user may want to filter by a specific label. You can also use label links to redirect users to a new page.
 
-You can also make labels clickable. A common use case for clickable labels is when a user may want to filter by a specific label, or may want to surface more information in a popover. You can also use label links to redirect users to a new page. Clickable labels have hover and focus states.
+<img src="./img/clickable-label.png" alt="Example of a clickable label" width="220"/>
 
-<img src="./img/clickable-label.png" alt="Example of a clickable label" width="219"/>
+### When to use editable labels
+Use editable labels when you want to allow users to organize and select resources manually. Adding editable labels enables the user to query for objects that have similar, dissimilar, or overlapping labels.
+
+<img src="./img/editable-label-feature.png" alt="Example of a editable label" width="165"/>
 
 ## Labels in context
 Labels can be used almost anywhere in a UI, but are commonly used in table views and catalogs.
 
 <img src="./img/table-view.png" alt="Example labels in a table" width="1209"/>
 
 <img src="./img/catalog-view.png" alt="Example labels in a catalog" width="1204"/>
+
+If you want to show a count, use a [badge](https://www.patternfly.org/v4/components/badge) instead.
```

Comment:
Thinking it might be best to move this up to the top of the guidelines, and append it to the end of the intro sentences, because it would probably get lost here.

Reply:
This has been added to the intro paragraph, makes sense that it's there versus the bottom. Thanks.